demisto_sdk/commands/content_graph/parsers/content_item.py

```diff
@@ -88,6 +92,9 @@ def __init__(
         super().__init__(path)
         self.relationships: Relationships = Relationships()
         self.git_sha: Optional[str] = git_sha
+        self.structure_errors: Optional[
+            List[StructureError]
+        ] = None  # TODO - remove when all ST validations are done
```

Comment:
let's make a list of these, so if we merge now, we don't forget about them

Reply:
I think at the meantime we solved all TODOs